src/frontend/packages/marsha-config/package.json

```diff
@@ -8,6 +8,7 @@
     "./tsconfig/*"
   ],
   "scripts": {
+    "start": "node index.js",
```

Comment:
FMI, what is the purpose of start here ?

Reply:
There is not a strong purpose, it is more like a shortcut if I need to run the script.